pkg/minikube/tunnel/kic/ssh_conn.go

```diff
@@ -142,14 +145,21 @@ func (c *sshConn) startAndWait() error {
 		return err
 	}
 
+	c.activeConn = true
 	// we ignore wait error because the process will be killed
 	_ = c.cmd.Wait()
 
+	// Wait is finished for connection, mark false.
+	c.activeConn = false
+
 	return nil
 }
 
 func (c *sshConn) stop() error {
-	out.Step(style.Stopping, "Stopping tunnel for service {{.service}}.", out.V{"service": c.service})
-
-	return c.cmd.Process.Kill()
+	if c.activeConn {
```

Comment:
can we set `c.activeConn` to `false` here? If somehow we call `c.stop()` twice, we'll try to kill already stopped process.